extensions/amp-nexxtv-player/amp-nexxtv-player.md

```diff
@@ -88,6 +88,21 @@ Indicates the source from which the embedded domain media is played. By default
 
 Ads are enabled by default. Set value to 1 to disable.
 
+##### autoplay (optional)
```

Comment:
let's remove nexxtv from this. their integration is broken: https://github.com/ampproject/amphtml/issues/8264